### ISSUE TITLE
Can't use node built in modules (fs, path, etc) in config files

### ISSUE THREAD

**Issue by snaptopixel** [tailwindcss/tailwindcss#1119]
@serh11p's [fix](https://github.com/tailwindcss/tailwindcss/commit/4a15678abb92676df08baef6f602a5f7a6869ab9) for watching modules in config files causes errors when require'ing node builtins like fs, path, etc.

```
(node:55727) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'fs'
    at Object.openSync (fs.js:448:3)
    at Object.readFileSync (fs.js:348:35)
    at createModule (node_modules/tailwindcss/lib/lib/getModuleDependencies.js:19:30)
    at mdl.requires.forEach.dep (node_modules/tailwindcss/lib/lib/getModuleDependencies.js:42:25)
    at Array.forEach (<anonymous>)
    at getModuleDependencies (node_modules/tailwindcss/lib/lib/getModuleDependencies.js:34:18)
    at node_modules/tailwindcss/lib/lib/registerConfigAsDependency.js:20:40
```

**Comment by serh11p** [comment]
Really!
Thanks for the issue.
I will fix it and create PR at morning!
<3

**Comment by adamwathan** [comment]
Resolved by #1121 👍
